source/_site_footer.erb

```diff
@@ -47,4 +47,9 @@
       </a>
     </div>
   </div>
+  <script type="text/javascript">
```

Comment:
We could lose the `type` from all of our script tags as the default is javascript.. see 'type' under https://developer.mozilla.org/en/docs/Web/HTML/Element/script